dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/newsession/NewSessionPresenter.java

```diff
@@ -27,6 +27,7 @@
 import com.spectralogic.dsbrowser.gui.services.tasks.CreateConnectionTask;
 import com.spectralogic.dsbrowser.gui.util.*;
 import com.spectralogic.dsbrowser.util.GuavaCollectors;
+import com.sun.org.apache.xpath.internal.operations.Bool;
```

Comment:
I think this is being included by accident.

Reply:
good catch